util/cgroup/cgroup_cpu.go

```diff
@@ -67,3 +67,12 @@ func getCgroupCPU(root string) (CPUUsage, error) {
 
 	return res, nil
 }
+
+// CPUShares returns the number of CPUs this cgroup can be expected to
+// max out. If there's no limit, NumCPU is returned.
+func (c CPUUsage) CPUShares() float64 {
+	if c.Period <= 0 || c.Quota <= 0 {
+		return float64(c.NumCPU)
```

Comment:
If this NumCPU is cgroup or contaner awarness?

Reply:
if c.Quota <= is true, it should be in physical machine. it just return NumCPU.